pycbc/strain.py

```diff
@@ -39,7 +39,7 @@
 from scipy.signal import kaiserord
  
 
-next_power_of_2 = lambda n: 2 ** (math.ceil(math.log(n, 2)) + 1)
+next_power_of_2 = lambda n: 1<<(n).bit_length()
```

Comment:
That guarantees that the result is an int, and also gives more correct results (try plotting both versions as a function of n). To me it also looks clearer, but that might be personal taste :)